src/NzbDrone.Core/Indexers/Definitions/Xthor/Xthor.cs

```diff
@@ -0,0 +1,470 @@
+                                Title = torrent.Name,
+                                Seeders = torrent.Seeders,
+                                Peers = torrent.Seeders + torrent.Leechers,
+                                MinimumRatio = 1,
+                                MinimumSeedTime = 345600,
+                                PublishDate = publishDate,
+                                Size = torrent.Size,
+                                Grabs = torrent.Times_completed,
+                                Files = torrent.Numfiles,
+                                UploadVolumeFactor = 1,
+                                DownloadVolumeFactor = torrent.Freeleech == 1 ? 0 : 1,
+                                Guid = guid,
+                                InfoUrl = details,
+                                DownloadUrl = link,
+                                TmdbId = torrent.Tmdb_id
+                            };
+
+                            return release;
+                        }));
+                }
+            }
+
+            return torrentInfos.ToArray();
+        }
+
+        private void CheckApiState(XthorError state)
+        {
+            // Switch on state
+            switch (state.Code)
+            {
+                case 0:
+                    // Everything OK
+                    break;
+                case 1:
+                    // Passkey not found
+                    throw new Exception("Passkey not found in tracker's database");
+                case 2:
+                    // No results
+                    break;
+                case 3:
+                    // Power Saver
+                    break;
+                case 4:
+                    // DDOS Attack, API disabled
+                    throw new Exception("Tracker is under DDOS attack, API disabled");
+                case 8:
+                    // AntiSpam Protection
+                    throw new Exception("Triggered AntiSpam Protection, please delay your requests !");
+                default:
+                    // Unknown state
+                    throw new Exception("Unknown state, aborting querying");
+            }
+        }
+
+        public Action<IDictionary<string, string>, DateTime?> CookiesUpdater { get; set; }
+    }
+
+    public class XthorSettingsValidator : AbstractValidator<XthorSettings>
+    {
+    }
+
+    public class XthorSettings : IIndexerSettings
+    {
+        private static readonly XthorSettingsValidator Validator = new XthorSettingsValidator();
+
+        public XthorSettings()
+        {
+            BaseUrl = "https://api.xthor.tk/";
+            Passkey = "";
+            FreeleechOnly = false;
+            Accent = 0;
+            NeedMultiReplacement = false;
+            MultiReplacement = "";
+            SubReplacement = "";
+            EnhancedAnime = true;
+            ByPassPageForTmDbid = true;
+            MaxPages = 1;
+        }
+
+        [FieldDefinition(1,
```

Comment:
single line, see other c# for examples; also adjust help text to be standardized
https://github.com/Prowlarr/Prowlarr/commit/0ddc530dd4b374980436eed15ed4fb0ac88b80ea

Reply:
Fixed